packages/sitecore-jss-proxy/src/middleware/healthcheck/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Router, Request, Response } from 'express';
+
+/**
+ * Middleware to handle health check requests
+ */
+const healthCheckMiddleware = () => (_req: Request, res: Response): void => {
```

Comment:
We can simplify that don't wrap it with additional function, right? Since it doesn't accept any configuration